src/ProcessManager/WebServerReadinessProbeTrait.php

```diff
@@ -36,23 +36,46 @@ private function checkPortAvailable(string $hostname, int $port, bool $throw = t
         }
     }
 
-    public function waitUntilReady(Process $process, string $url, bool $ignoreErrors = false): void
+    public function waitUntilReady(Process $process, string $url): void
     {
-        $context = \stream_context_create(['http' => [
```

Comment:
Why this change? It will not work if the `/` route isn't defined for instance.

Reply:
Because it instead uses a socket, I thought it might be a much better idea

Reply:
I'm 👎 to use a socket. Using a HTTP client makes sure that the underlying API speaks HTTP. (A server supporting another protocol may already use this port).